nevergrad/optimization/test_base.py

```diff
@@ -152,3 +155,25 @@ def test_naming() -> None:
     np.testing.assert_equal(
         repr(opt), f"Instance of BlubluOptimizer(parametrization={instru_str}, budget=4, num_workers=1)"
     )
+
+
+class MinStorageFunc:
+    """Stores the minimum value obtained so far"""
+
+    def __init__(self) -> None:
+        self.min_loss = float("inf")
+
+    def __call__(self, score: int) -> float:
+        self.min_loss = min(score, self.min_loss)
+        return score
+
+
+def test_recommendation_correct() -> None:
```

Comment:
A possible test would be the example in the issue.
That said, I find the fix quite cool and convincing.

Reply:
Actually that is the same test, just simplified a bit, but it does exactly the same